File: 1768.cpp
#include<iostream>
#include<string>

using namespace std;

class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string ans = "";
        int n = min(word1.size(), word2.size());
        for(int i = 0; i < n; i++){
            ans += word1[i];
            ans += word2[i];
        }
        return ans + word1.substr(n) + word2.substr(n);
    }
};

int main(){
    string a, b;
    cin >> a >> b;
    Solution s;
    cout << s.mergeAlternately(a, b) << endl;
}

File: 1768.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        return ''.join(a + b for a, b in zip_longest(word1, word2, fillvalue=''))
    
if __name__ == '__main__':
    s = Solution()
    print(s.mergeAlternately(input(), input()))